class02/homework2.py

```diff
@@ -0,0 +1,395 @@
+import os.path
+import csv
+import json
+
+'''
+Edit this file to complete Exercise 5
+'''
+
+# 1. Write a Python program to find those numbers which
+# are divisible by 7 and multiple of 5, between 1500 and 2700 (both included).
+
+# code up your solution here
+
+
+def question1():
+    for i in range(1500, 2701):
+        if i % 7 == 0 and i % 5 == 0:
+            print(i)
+
+
+question1()
+
+
+# 2. Write a Python program to count the number of
+# even and odd numbers from a series of numbers.
+# example: numbers = (1, 2, 3, 4, 5, 6, 7, 8, 9)
+# output:
+# >>> Number of even numbers : 4
+# >>> Number of odd numbers : 5
+# code up your solution here
+
+
+def question2():
+    numbers = (1, 2, 3, 4, 5, 6, 7, 8, 9)
+    print('Number of even numbers:', len([i for i in numbers if i % 2 == 0]))
+    print('Number of odd numbers:', len([i for i in numbers if i % 2 != 0]))
+
+
+question2()
+
+
+# 3. Write a Python program which iterates the integers
+# from 0 to 50. For multiples of three print "Fizz"
+# instead of the number and for the multiples of five
+# print "Buzz". For numbers which are multiples of both
+# three and five print "FizzBuzz".
+
+# Expected Output :
+# >>> fizzbuzz
+# >>> 1
+# >>> 2
+# >>> fizz
+# >>> 4
+# >>> buzz
+# >>> ...
+
+# code up your solution here
+def question3():
```

Comment:
This problem is not giving the expected answer - for example, 0, 15, and 30 should be 'FizzBuzz'. Try and see if you can fix it.